### PR TITLE
Support kong migration

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

File: app/controllers/CommercialForm.scala [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
 import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class CommercialForm(dynamo: DB, kong: Kong, awsEmail: MailClient, val messagesApi: MessagesApi) extends Controller with I18nSupport {
+class CommercialForm(dynamo: DB, awsEmail: MailClient, val messagesApi: MessagesApi) extends Controller with I18nSupport {
   import CommercialForm._
   import Forms.CommercialRequestKeyFormData
 
-  private val logic = new CommercialFormLogic(dynamo, kong)
+  private val logic = new CommercialFormLogic(dynamo)
```
Comment (LATaylor-guardian) [inline app/controllers/CommercialForm.scala]: Kong was passed into this class but never used so I removed it.

**Comment by cb372** [comment]
👍  once the tests pass.

Don't forget to update the CODE and PROD config files before deploying. Probably worth manually testing in CODE as well.

**Comment by LATaylor-guardian** [comment]
Updated CODE config, manually tested and working fine. Will now do the same for PROD.

**Comment by LATaylor-guardian** [comment]
Updated PROD config. Merging.

**Pull request merged**